### DISCUSSION TITLE
Iterate through multiple BMSes

### DISCUSSION THREAD

**Discussion by dan-cristian** [syssi/esphome-jk-bms#281] [Ideas]
I have many battery packs/BMSes and I don't really need a very frequent read on BMS values.
I would like to use one single ESP32 board to iterate through multiple BMS boards so I reduce the number of ESP boards I currently use (9 ESPs). I only need the MQTT topic prefix to change when connecting to a BMS board, I currently use board MAC as prefix.

**Comment by torsteinelv** [comment]
Is this not easy to just add in the loop?

**Reply by syssi** [reply]
Cross-Post: https://github.com/syssi/esphome-jbd-bms/discussions/32
